res/controllers/Behringer-CMD-Micro-scripts.js

```diff
@@ -0,0 +1,96 @@
+////////////////////////////////////////////////////////////////////////
+// JSHint configuration                                               //
+////////////////////////////////////////////////////////////////////////
+/* global engine                                                      */
+/* global script                                                      */
+/* global print                                                       */
+/* global midi                                                        */
+////////////////////////////////////////////////////////////////////////
+(function(global) {
+    var BCM = {
+        DownButtons: [0, 0],
+        PitchReset: [false, false]
+    };
+
+    var SCRATCH_ALPHA = 1.0/8;
+    var SCRATCH_BETA  = SCRATCH_ALPHA/32;
+    var SCRATCH_RPM   = 33 + (1.0/3);
+
+    var touchFunc = function(forChannel) {
+        return function(channel, control, value, status, group) {
+            if (0x90 === (status & 0xF0)) {
+                engine.scratchEnable(forChannel,
+                                     128,
+                                     SCRATCH_RPM,
+                                     SCRATCH_ALPHA,
+                                     SCRATCH_BETA,
+                                     true);
+            } else {
+                engine.scratchDisable(forChannel, true);
+            }
+        };
+    };
+
+    var tickFunc = function(forChannel) {
+        return function(channel, control, value, status, group) {
```

Comment:
My guess is that he doesn't realize that the channel is passed in by mixxx and does not have to be bound manually. Same reason why there are `leftWheelTick` and `rightWheelTick` bindings instead of just `wheelTick` that takes advantage of channel.